.github/workflows/security.yml

```diff
@@ -0,0 +1,28 @@
+on: 
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**/*.md"
+  schedule:
+    - cron: 15 14 * * 6
+
+jobs:
+  veracode-sca-task:
+    runs-on: ubuntu-latest
+    name: Scan repository for Issues
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run Veracode SCA
+        env:
+          SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+        uses: veracode/veracode-sca@v1.09
```

Comment:
Does it scan dependencies as well?

Reply:
It does: see https://docs.veracode.com/r/c_sc_what_is